core/src/main/java/org/infinispan/transaction/impl/TransactionTable.java

```diff
@@ -277,37 +277,41 @@ public int getMinTopologyId() {
    }
 
    public void cleanupLeaverTransactions(List<Address> members) {
-      // Can happen if the cache is non-transactional
-      if (remoteTransactions == null)
-         return;
+      try {
+         // Can happen if the cache is non-transactional
+         if (remoteTransactions == null)
```

Comment:
I see what you mean now... I was confused about the "move" part, because there was no `try` block before :)